backend/danswer/background/update.py

```diff
@@ -298,10 +298,13 @@ def _index(
                 net_doc_change += new_docs
                 chunk_count += total_batch_chunks
                 document_count += len(doc_batch)
+
+                # This new value is updated every batch, so UI can refresh per batch update
                 update_docs_indexed(
                     db_session=db_session,
                     index_attempt=attempt,
-                    num_docs_indexed=document_count,
```

Comment:
num_docs_indexed is total docs touched/refreshed, on frontend we were showing new docs